docs/intro/arch_overview/load_balancing.rst

```diff
@@ -128,3 +128,156 @@ with regard to percentage relations in the local zone between originating and up
   In this case the local zone of the upstream cluster can get all of the requests from the
   local zone of the originating cluster and also have some space to allow traffic from other zones
   in the originating cluster (if needed).
+
+.. _arch_overview_load_balancer_subsets:
+
+Load Balancer Subsets
+---------------------
+
+Envoy may be configured to divide hosts within an upstream cluster into subsets based on metadata
+attached to the hosts. Routes may then specify the metadata that a host must match in order to be
+selected by the load balancer, with the option of falling back to a predefined set of hosts,
+including any host.
+
+Subsets use the load balancer policy specified by the cluster. The original destination policy may
+not be used with subsets because the upstream hosts are not known in advance. Subsets are compatible
+with zone aware routing, but be aware that the use of subsets may easily violate the minimum hosts
+condition described above.
+
+If subsets are configured_ and a route specifies no metadata or no subset matching the metadata
+exists, the subset load balancer initiates its fallback policy. The default policy is
+``NO_ENDPOINT``, in which case the request fails as if the cluster had no hosts. Conversely, the
+``ANY_ENDPOINT`` fallback policy load balances across all hosts in the cluster, without regard to
+host metadata. Finally, the ``DEFAULT_SUBSET`` causes fallback to load balance among hosts that
+match a specific set of metadata.
+
+.. _configured: https://github.com/envoyproxy/data-plane-api/blob/9897e3f/api/cds.proto#L237
```

Comment:
I'm not familiar with this construct in sphinx. What does this render?

Reply:
An external link at `configured_` in the previous paragraph. The construct for internal links wasn't actually rendering correctly. That said, looking at more than just the RST cheatsheet, it looks like there's a way to inline it. I'll see if I can get that t work.

Reply:
It's fine either way. Was just curious.